pkg/platformconfig/reader.go

```diff
@@ -83,5 +83,9 @@ func (r *Reader) GetDefaultConfiguration() *Config {
 				{Level: "debug", Sink: "stdout"},
 			},
 		},
+		ScaleToZero: ScaleToZero{
+			ScalerInterval:           "1m",
+			ResourceReadinessTimeout: "2m",
```

Comment:
This might have adverse effects? What're the defaults today? does this change them? (@Hedingber wdyt)

Reply:
They are being used when initializing resource scaler (pkg/platform/kube/resourcescaler/resourcescaler.go:134)

If *no* platform configuration path if provided, it is being initialized with empty string and hence causing to an `err` (default platform configuration is broken up until this fix)

Alternatively, it reads the platform configuration and use it as is (which is not the case here)

Reply:
So this does not effect k8s helm use case?
👍

Reply:
Decided to move it to resourcescaler package, to handle all scenarios
- k8s helm _with out_ platform configuration -> fixed
- k8s helm _with_ platform configuration, no "scalerInterval/ResourceReadinessTimeout" while autoscaler/dlx services are enabled -> fixed 
- k8s helm with platform configuration, "scalerInterval/ResourceReadinessTimeout" is provided -> ok